micronota/util.py

```diff
@@ -102,9 +115,9 @@ def _download(src, dest, **kwargs):
 
 @contextmanager
 def _tmp_file(*args, **kwargs):
```

Comment:
i thought you could have them persist?

On Wed, Feb 24, 2016 at 10:35 PM, Zech Xu <notifications@github.com> wrote:

> In micronota/util.py
> <https://github.com/biocore/micronota/pull/46#discussion_r54054145>:
>
> > @@ -102,9 +115,9 @@ def _download(src, dest, **kwargs):
> >
> >  @contextmanager
> >  def _tmp_file(*args, **kwargs):
>
> because the file will be deleted immediately when it is closed, which
> seemed a deal breaker. let me rethink about it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/micronota/pull/46/files#r54054145>.
>